src/repositories/user.repository.ts

```diff
@@ -11,13 +11,18 @@ import { defaults } from 'lodash';
 import * as Chance from 'chance';
 
 export class UserRepository extends Repository {
-  async info(id: string | number): Promise<UserRepositoryInfoResponseUser> {
+  async infoByUserId(id: number): Promise<UserRepositoryInfoResponseUser> {
```

Comment:
This has to stay `info` as methods should be named after the endpoint (which is `/info`) and in addition we can't rename it due to compatibility reasons. Also the argument may be a `string` if the id is too big for a `number`.

```suggestion
  async info(id: string | number): Promise<UserRepositoryInfoResponseUser> {
```